js/components/ChartSummary.vue

```diff
@@ -1,16 +1,13 @@
 <template>
-  <div>
-    <div>
+  <div class="row">
+    <div class="col-md-6">
       <table ref="totalsTable" class="table table-striped table-bordered">
         <caption>
           {{ messages.totalsCaption }}
-          <button class="copy-link pull-right btn btn-link btn-sm" @click="copyTable($refs.totalsTable)">
-            <span
-              class="glyphicon glyphicon-copy"
-              aria-hidden="true"
-              :title="messages.copyTable">
-            </span>
-          </button>
+          <a href="#" class="copy-link pull-right float-right" data-description="reload" role="button" @click.prevent="copyTable($refs.totalsTable)">
```

Comment:
Is this the right `data-description`? I'm not following reload on the copy table button.

Reply:
Good catch. I think this was either a copy-paste error or leftover from a previous iteration. We're not actually using the data attribute anymore so I'll just remove it.